tests/db_integrations/test.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: test
+  hosts: ofn_servers
+  remote_user: "{{ user }}"
+
+  tasks:
+    - name: test | db_integrations | expect postgresql.conf to be updated
+      command: grep -q "listen_addresses = '*'" {{ postgresql_config_path }}/postgresql.conf
```

Comment:
I'm not sure how the PostgreSQL configuration playbook updates the config files. Don't we need to match the _start of the line_ here to make sure that the setting is not commented out?

Reply:
That might be an improvement, yes. It works currently because the initial setup of the test server applies the defaults, which don't contain the string `listen_addresses = '*'`.

Reply:
We apply this non-default setting in the setup step for the test, in `tests/db_integrations/setup.yml` 
https://github.com/openfoodfoundation/ofn-install/pull/421/files#diff-2efee1d210c2ed1f315a9ea78be94230